webhooks/external_account_config.go

```diff
@@ -70,12 +69,11 @@ func NewExternalAccountCredentials(aud, gsaEmail string) *ExternalAccountCredent
 	return creds
 }
 
-func (e *ExternalAccountCredentials) String() string {
+func (e *ExternalAccountCredentials) Render() (string, error) {
 	b, err := json.MarshalIndent(e, "", "  ")
```

Comment:
nits: perhaps we don't need an indent here? Indentation is just for readability, right?

Reply:
I made the indentation optional.